### PR TITLE
README.md: clarify that Lima is similar to WSL2 but not similar to WSL1

### PR DESCRIPTION
**Pull request by AkihiroSuda**

### CODE REVIEW

**Comment by afbjorklund** [comment]
@AkihiroSuda : I learned that WSL2 is more similar to something like LXD, I think it runs a _single_ shared Hyper-V VM and then does "system containers" for the distributions ? So while it is still more similar to WSL2 than WSL1, not sure "equivalent" - maybe "similar to" ?

**Comment by AkihiroSuda** [comment]
Thanks, updated the wording

File: README.md [hunk 1]
```diff
@@ -3,9 +3,9 @@
 
 # Lima: Linux virtual machines (on macOS, in most cases)
 
-Lima launches Linux virtual machines with automatic file sharing, port forwarding, and [containerd](https://containerd.io).
+Lima launches Linux virtual machines with automatic file sharing, port forwarding (similar to WSL2), and [containerd](https://containerd.io).
```
Comment (jandubois) [inline README.md:6]: Just nit picking, but I would maybe word it like this, to make it clear that both file sharing and port forwarding are similar to WSL2. But I'm also fine with leaving it as-is.

```suggestion
Lima launches Linux virtual machines with automatic file sharing and port forwarding (similar to WSL2), and [containerd](https://containerd.io).
```

**Review by jandubois — APPROVED** [review]
Thanks!

File: README.md (hunk 1 above)
Comment (AkihiroSuda) [reply README.md]: Thanks, updated

**Pull request merged**